core/services/feeds/models.go

```diff
@@ -167,6 +166,9 @@ const (
 	// but cancelled by the node op. A cancelled spec is not being run by the
 	// node.
 	SpecStatusCancelled SpecStatus = "cancelled"
+	// SpecStatusDeleted defines a spec status which was deleted by the FMS.
+	// A deleted job spec is not being run by the node.
+	SpecStatusDeleted SpecStatus = "deleted"
```

Comment:
I don't believe you need to add a spec status for deleted. It is only the job proposal that is deleted